test/setup/platforms/kind.ts

```diff
@@ -7,7 +7,7 @@ const clusterName = 'kind';
 
 export async function setupTester(): Promise<void> {
   const osDistro = platform();
-  await download(osDistro);
+  await download(osDistro, 'v0.8.1');
```

Comment:
why did you remove the version here? this parameter looks like mandatory in `download`

Reply:
No no, added a version here!